libraries/s3/lib_test/test_async.ml

```diff
@@ -3,11 +3,13 @@ open Aws_s3_test
 module T = TestSuite (struct
   type 'a m = 'a Async.Deferred.t
 
-  let access_key = Unix.getenv "AWS_ACCESS_KEY"
+  let access_key = Unix.getenv "AWS_ACCESS_KEY_ID"
 
-  let secret_key = Unix.getenv "AWS_SECRET_KEY"
+  let secret_key = Unix.getenv "AWS_SECRET_ACCESS_KEY"
```

Comment:
Happy to change these values but could it be done in another PR and the change made across the board for all `lib_test` tests.